app/src/main/java/com/celzero/bravedns/NonStoreAppUpdater.kt

```diff
@@ -51,15 +51,16 @@ class NonStoreAppUpdater(val baseURL:String, private val persistentState: Persis
                     //creating json object
                     val jsonObject = JSONObject(stringResponse)
                     val responseVersion = jsonObject.getInt("version")
-                    val updateValue = jsonObject.getBoolean("update")
+                    val shouldUpdate = jsonObject.getBoolean("update")
```

Comment:
prefer the [`opt` variants](https://developer.android.com/reference/kotlin/org/json/JSONObject?hl=en#opt(kotlin.String)) of the `JSONObject`.

Reply:
Modified as above.